pkg/graph/schema.resolvers.go

```diff
@@ -488,17 +489,21 @@ func (r *mutationResolver) DeleteAccessibilityRequestDocument(ctx context.Contex
 }
 
 func (r *mutationResolver) UpdateAccessibilityRequestStatus(ctx context.Context, input *model.UpdateAccessibilityRequestStatus) (*model.UpdateAccessibilityRequestStatusPayload, error) {
+	requesterEUAID := appcontext.Principal(ctx).ID()
```

Comment:
Also maybe not in scope for this story: What's the authorization permissions on this mutation? Is it 508 Team only?

Reply:
We have different ways of adding auth to endpoints depending on what the auth is. Talk to me if you want advice

Reply:
@Igarfinkle Yes it should be 508 team only.

Reply:
Oh great, all we need to do then is add @hasRole(role: EASI_508_TESTER_OR_USER) to the mutation in the schema.